docs/UserGuide.md

```diff
@@ -83,19 +83,22 @@ You can find out more about the command terminologies here:
 
 ### Adding an assignment: `add`
 
-Adds an assignment into your schedule.
+Format: `add n/NAME_OF_ASSIGNMENT d/DEADLINE_OF_ASSIGNMENT TIME_ASSIGNMENT_IS_DUE mod/MODULE​[priority/PRIORITY] 
+[remind]`
 
-Format: `add n/NAME_OF_ASSIGNMENT d/DEADLINE_OF_ASSIGNMENT TIME_ASSIGNMENT_IS_DUE mod/MODULE​ [remind]`
+You can add your assignments into your schedule so that you can manage all your assignments and
+ academic tasks conveniently. 
 
 <div markdown="span" class="alert alert-success">
-   
+
 **:bulb: Tip:**
 You can include `remind` when adding an assignment instead of using the `remind` command to set reminders after adding an assignment.
 </div>
 
 Examples:
-* `add n/Lab report 3 d/23-04-2020 1230 mod/CS2100`
-* `add n/Tutorial 2 d/29-06-2020 1400 mod/CS2100 remind`
+* To add a lab report assignment that is **due** on 23th April 2020 12.30pm and **associated** with the module CS2100, you can 
+simply enter `add n/Lab report d/23-04-2020 1230 mod/CS2100`.
+* To add a CS1010S tutorial assignment that is of **high priority**, you can enter `add n/Tutorial 2 d/21-04-2020 2359 mod/CS1010S priority/HIGH`
```

Comment:
Looks good :) However, as the rest of us put the descriptions of the examples outside of the `Example: ` section, it may be good to write the examples as their own paragraphs (and add some diagrams if needed) for consistency. Then under `Examples:` section, you may either write some examples commands (without the description) or just remove the section if you believe it is not necessary.

Also, another note is that @hyngkng has changed the priority prefix from `priority/` to `p/`

Reply:
okay thanks!